risc0/cargo-risczero/build.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright 2023 RISC Zero, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::env;
+
+#[cfg(feature = "experimental")]
+mod runtime {
+    use std::{env, fs, io, path::Path};
+
+    use risc0_build::build_rust_runtime;
+    use zip::{write::FileOptions, CompressionMethod, ZipWriter};
+
+    pub fn build_and_zip_runtime() {
```

Comment:
It seems like we should make this library part of the toolchain, rather than part of `cargo-risczero`